ml-libs/CMakeLists.txt

```diff
@@ -6,6 +6,54 @@ if(THEROCK_ENABLE_MIOPEN)
     list(APPEND optional_profiler_deps roctracer rocprofiler-sdk)
   endif()
 
+  ##############################################################################
+  # Composable_kernel
+  ##############################################################################
+
+  therock_cmake_subproject_declare(composable_kernel
```

Comment:
We'll want to make this declaration conditional on "not Windows". There are some other examples of that like here: https://github.com/ROCm/TheRock/blob/b3af08a5cc0eceb589d02291585e4c4d46c2613d/math-libs/BLAS/CMakeLists.txt#L47-L53